Citus Replica Consistency Issue

### Description

Acceptance tests in mainnet staging show the following failure:
```

2024-07-10 10:45:57.570 | 2024-07-10T15:45:57.567Z ERROR ForkJoinPool-1-worker-3 c.h.m.t.e.a.s.EthereumFeature Scenario failure:  I successfully created a signer account with an EVM address alias validate the signer account and its balance I successfully create contract by Legacy ethereum transaction the mirror node REST API should return status 200 for the eth contract creation transaction the mirror node contract results API should return an accurate gas consumed the mirror node REST API should verify the deployed contract entity by eth call <-  Expecting actual not to be null I successfully call function using EIP-1559 ethereum transaction the mirror node REST API should return status 200 for the ethereum transaction the mirror node contract results API should return an accurate gas consumed the mirror node REST API should verify the ethereum called contract function I successfully call function using EIP-2930 ethereum transaction the mirror node REST API should return status 200 for the ethereum transaction the mirror node contract results API should return an accurate gas consumed the mirror node REST API should verify the ethereum called contract function  java.lang.AssertionError:  Expecting actual not to be null 	at com.hedera.mirror.test.e2e.acceptance.steps.BaseContractFeature.verifyContractExecutionResultsByTransactionId(BaseContractFeature.java:87) 	at com.hedera.mirror.test.e2e.acceptance.steps.EthereumFeature.verifyDeployedContractMirror(EthereumFeature.java:135) 	at ✽.the mirror node REST API should verify the deployed contract entity by eth call(classpath:features/contract/ethereum.feature:13)
```

This failure is because the block info was not present in the response. This shouldn't be possible as we only grab the block info for the returned contract results. However, the block info query is done as a separate query once we have obtained the relevant timestamps from the contract results. record_file is just a normal postgres table so in citus it only exists on the coordinator. It appears that the request to get block info went to the coordinator replica and the data was not replicated there yet. 

```

2024-07-10 10:45:57.168 | 2024-07-10 15:45:57.168 UTC [439] LOG C-0xa13670: mirror_node/mirror_rest@127.0.0.1:44810 login attempt: db=mirror_node user=mirror_rest tls=no |  
2024-07-10 10:45:57.168 | 2024-07-10 15:45:57.168 UTC [439] LOG C-0xa11b90: mirror_node/mirror_rest@127.0.0.1:44814 login attempt: db=mirror_node user=mirror_rest tls=no

2024-07-10 10:45:53.738 | 2024-07-10 15:45:53 UTC [8984]: db=,user=,app=,client= DETAIL:  Last completed transaction was at log time 2024-07-10 15:45:53.305359+00.

```

### Steps to reproduce

This is a race condition

### Additional context

_No response_

### Hedera network

other

### Version

v0.109.0

### Operating system

None

--- a/charts/hedera-mirror/templates/stackgres/stackgres-cluster.yaml
+++ b/charts/hedera-mirror/templates/stackgres/stackgres-cluster.yaml
@@ -40,7 +40,7 @@ spec:
     version: {{ .Values.stackgres.postgresVersion | quote }}
   prometheusAutobind: {{ or .Values.stackgres.coordinator.enableMetricsExporter .Values.stackgres.worker.enableMetricsExporter }}
   replication:
-    mode: async  # We might consider setting coordinator to sync or the proposed sync-all in the future
+    mode: {{ .Values.stackgres.coordinator.replication.mode }}
   shards:
     clusters: {{ .Values.stackgres.worker.instances }}
     configurations:

--- a/charts/hedera-mirror/values.yaml
+++ b/charts/hedera-mirror/values.yaml
@@ -294,6 +294,8 @@ stackgres:
       work_mem: "32MB"
     enableMetricsExporter: false
     instances: 1
+    replication:
+      mode: sync-all
     resources:
       cpu: 100m
       memory: 1Gi

